Adyen/validation.py

```diff
@@ -16,17 +16,66 @@
 actions['refund'] = ["modificationAmount", "originalReference"]
 actions['cancelOrRefund'] = ["originalReference"]
 
+payout_required_fields = {
+    'confirmThirdParty': (
+        'merchantAccount',
+        'originalReference'
+    ),
+    'declineThirdParty': (
+        'merchantAccount',
+        'originalReference'
+    ),
+    'storeDetail': (
+        'merchantAccount',
+        'bank.iban',
+        'bank.ownerName',
+        'bank.countryCode',
+        'recurring.contract',
+    ),
+    'submitThirdParty': (
+        'amount.currency',
+        'amount.value',
+        'merchantAccount',
+        'recurring.contract',
+        'reference',
+        'shopperEmail',
+        'shopperReference',
+        'selectedRecurringDetailReference'
+    ),
+    'storeDetailAndSubmitThirdParty': (
+        'merchantAccount',
+        'shopperEmail',
+        'shopperReference',
+        'reference',
+        'recurring.contract',
+        'amount.currency',
+        'amount.value',
+        'bank.iban',
+        'bank.ownerName',
+        'bank.countryCode',
```

Comment:
Can you please remove the bank details from the validation?
If card details are provided, bank details are no longer mandatory.